projects/demo/src/modules/directives/skeleton/examples/2/index.html

```diff
@@ -0,0 +1,33 @@
+<p>
+    You can pass
+    <code>boolean</code>
+    to toggle skeleton for elements. For multi line text you can use
+    <code>string</code>
+    to serve as a placeholder underneath the text skeleton or a
+    <code>number</code>
+    to generate this many random sized words, while your actual text is loading
+</p>
```

Comment:
Let's move this long text to description of this example.
It overcomplicates this code-tab.

```html
<tui-doc-example
    id="text"
    heading="Text"
    [content]="2 | tuiExample: 'html'"
    [description]="textPlaceholderDescription"
>
    <ng-template #textPlaceholderDescription>
        You can pass
        <code>boolean</code>
        to toggle skeleton for elements. For multi line text you can use
        <code>string</code>
        to serve as a placeholder underneath the text skeleton or a
        <code>number</code>
        to generate this many random sized words, while your actual text is loading
    </ng-template>

    <tui-skeleton-example-2></tui-skeleton-example-2>
</tui-doc-example>
```